gluster_exporter/main.go

```diff
@@ -32,17 +31,79 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 	glusterMetrics = append(glusterMetrics, glusterMetric{name: name, fn: fn, intervalSeconds: time.Duration(intervalSeconds)})
 }
 
-func getPeerID() string {
-	return *peerid
+func getWorkingDir( glusterMgmt *string ) string {
+        if( *glusterMgmt == "glusterd2" ) {
+                return "/var/lib/glusterd2"
```

Comment:
Do not hard code the path here, add new CLI flag. This path is configurable in both glusterd2 and glusterd1.

Reply:
if glusterdWorkdir != "" {
        return glusterdWorkdir
    }

    if( *glusterMgmt == "glusterd2" ) {
        return defaultGd2Workdir
    }
    return defaultGd1Workdir

Reply:
const (
        defaultGd1Workdir = "/var/lib/glusterd"
        defaultGd2Workdir = "/var/lib/glusterd2"
    )